src/config/dynamic-theme-fixes.config

```diff
@@ -5105,7 +5105,8 @@ INVERT
 img[src*="/static/media/logo-large"]
 img[src*="/static/media/samsung"]
 img[src*="/static/media/sonos"]
-a.d-none.d-md-inline
+.d-md-inline > img
+g.rsm-geographies
```

Comment:
Invert the map in the analytics page
![Map 1](https://user-images.githubusercontent.com/14346945/100694612-1e388c00-3398-11eb-852b-abfbc07eb30a.jpg)
![Map 2](https://user-images.githubusercontent.com/14346945/100694619-21cc1300-3398-11eb-8271-394b8c4010bb.jpg)

Reply:
It seems like the non-lighten country's is almost not visible. I think this Selector can be played around e.g. 
```CSS
g.rsm-geographies {
    filter: invert(1) hue-rotate(180deg) !important;
}
```
or
```CSS
g.rsm-geographies {
    filter: invert(50%) hue-rotate(180deg) !important;
}
```

Reply:
This looks a little brighter
```css
g.rsm-geographies {
    filter: invert(1) hue-rotate(180deg) !important;
}
```
![invert(1)](https://user-images.githubusercontent.com/14346945/100755569-fc212700-33f4-11eb-9f0c-7d17f567b07e.jpg)

Compared to normal invert
![INVERT](https://user-images.githubusercontent.com/14346945/100755794-3a1e4b00-33f5-11eb-944f-e8126e73dbde.jpg)


But this one is just completely gray
```css
g.rsm-geographies {
    filter: invert(50%) hue-rotate(180deg) !important;

}
```
![invert(50%)](https://user-images.githubusercontent.com/14346945/100755671-1c50e600-33f5-11eb-8637-6e253f2bcb01.jpg)

Reply:
I noticed something, If you increase Dark Reader's brightness slider, the map gets darker, and if you decrease the brightness it gets brighter.

You can see for yourself through this link [https://my.nextdns.io/start/ ](url) (no registration required)

You will find the map at the bottom of the Analytics tab.

Reply:
> This looks a little brighter
> 
> ```css
> g.rsm-geographies {
>     filter: invert(1) hue-rotate(180deg) !important;
> }
> ```

Let's use that then It seems in my eyes the best.

Reply:
I forgot to mention before that I had Dark Reader's brightness at 90% and contrast at 110%, and as I said before:
> If you increase Dark Reader's brightness slider, the map gets darker, and if you decrease the brightness it gets brighter

So actually it looks darker than normal invert when brightness and contrast are at 100%
![Screenshot (2020-12-01) (11-47-52 PM)](https://user-images.githubusercontent.com/14346945/100800851-d2d1bc80-342f-11eb-9c88-11ac41fde17b.jpg)

Reply:
Yeah because of the `invert` the values will be inversed as well and has an inversed effect.
However the ^ CSS should work on default settings and slightly changed settings.

Reply:
The last picture is on the default settings, and as you can see it's too dark.

Reply:
Hmm what about
```CSS
    filter: invert(1) hue-rotate(180deg) brightness(150%) !important;
```
I think messing around with the brightness filter can solve the problem.